### ISSUE TITLE
BUG: syscalls_file only works on 64-bit systems

### ISSUE THREAD

**Issue by pcmoore** [linux-audit/audit-testsuite#17]
The problem is the following line in the syscalls_file/test file:

```
$result = system("auditctl -a always,exit -F arch=b64 -S open -k $key");
```

... which results in the following error on 32-bit x86:

```
# auditctl -a always,exit -F arch=b64 -S open
arch elf mapping not found
```

**Comment by The-Mule** [comment]
I would take it if you don't mind. It is a low hanging fruit.

**Comment by pcmoore** [comment]
@The-Mule it's yours - thanks!

**Comment by pcmoore** [comment]
Fixed via 262c67c28e5eec0e623b400478aa300f364577a5.
